diff -uNr dansguardian-2.8.0.4.old/autoconf/linux.in dansguardian-2.8.0.4/autoconf/linux.in
--- dansguardian-2.8.0.4.old/autoconf/linux.in	2005-02-20 16:25:10.000000000 +0100
+++ dansguardian-2.8.0.4/autoconf/linux.in	2005-02-22 12:26:34.700489000 +0100
@@ -85,10 +85,6 @@
 	cp -f ./dansguardian.pl $I$(CGIBINLOCATION)dansguardian.pl
 	chmod o+x $I$(CGIBINLOCATION)dansguardian.pl
 	cp -f ./dansguardian.8.gz $I$(MANUALFILES)man8/dansguardian.8.gz
-	chown -R $(PROXYUSER):$(PROXYGROUP) $I$(LOGLOCATION)
-	chmod -R u+wr $I$(LOGLOCATION)
-	chmod -R og-w $I$(LOGLOCATION)
-	chmod u+wrx $I$(LOGLOCATION)
 	if [ -f $(CHKCONF) ]; then $(CHKCONFIG); fi
 	if [ -f $(DEB_RC) ]; then $(DEB_RC) dansguardian defaults 50; fi
 	if [ -d $(LOGROTATELOCATION) ]; then cp -f ./logrotate.dansguardian $I$(LOGROTATELOCATION)dansguardian; fi
