--- configure.old	2005-08-14 01:47:57.000000000 +0200
+++ configure	2005-12-03 11:54:05.000000000 +0100
@@ -45,6 +45,7 @@
 logdir='/var/log/dansguardian/'
 piddir='/var/run/'
 logrotatedir='/etc/logrotate.d/'
+datadir='/usr/share/dansguardian/'
 gccver=`gcc -dumpversion | sed '1,$s/.[0-9]*$//'`
 
 case `uname -s` in
@@ -2808,7 +2809,7 @@
 echo "# and easier to customise the access denied page." >>dansguardian.conf
 echo "# The language file is used no matter what setting however." >>dansguardian.conf
 echo "#" >>dansguardian.conf
-$extendedecho "languagedir = '$prefixdir$sysconfdir\c" >>dansguardian.conf
+$extendedecho "languagedir = '$prefixdir$datadir\c" >>dansguardian.conf
 echo "languages'" >>dansguardian.conf
 echo "" >>dansguardian.conf
 echo "# language to use from languagedir." >>dansguardian.conf
@@ -2875,8 +2876,8 @@
 echo "# 0 = off" >>dansguardian.conf
 echo "# 1 = on (default)" >>dansguardian.conf
 echo "usecustombannedimage = 1" >>dansguardian.conf
-$extendedecho "custombannedimagefile = '$prefixdir$sysconfdir\c" >>dansguardian.conf
-echo "transparent1x1.gif'" >>dansguardian.conf
+$extendedecho "custombannedimagefile = '$prefixdir$datadir\c" >>dansguardian.conf
+echo "pics/transparent1x1.gif'" >>dansguardian.conf
 echo "" >>dansguardian.conf
 echo "" >>dansguardian.conf
 echo "" >>dansguardian.conf
 
